api/STYLE.md

```diff
@@ -34,6 +34,9 @@ In addition, the following conventions should be followed:
   implementation. These indicate that the entity is not implemented in Envoy and the entity
   should be hidden from the Envoy documentation.
 
+* For extensions that are a work-in-progress or the documentation is hidden, set its `status`
```

Comment:
RE "documentation is hidden", I assume you mean the entire documentation is hidden (and not some field or sub-message).

Reply:
I noticed it for when the proto config has the "[#not-implemented-hide:]" annotation in the comments.  The proto also has the "[#extension: ...]" annotation, which I assume is where it gets correlated to the extension entry in `extensions_metadata.yaml`